expyfun/io/_parse.py

```diff
@@ -4,10 +4,43 @@
 
 import numpy as np
 import csv
+import ast
+import json
+
+
+def read_tab_raw(fname):
+    """Read .tab file from expyfun output without segmenting into trials.
+
+    Parameters
+    ----------
+    fname : str
+        Input filename.
+
+    Returns
+    -------
+    data : list of tuple
+        The data with each line from the tab file being a tuple in a list.
+        Each tuple is of the form (``timestamp``, ``key``, ``value``).
```

Comment:
`See Also` section with `read_tab` (and add the reverse to `read_tab` docstring)